files/js-cookie/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "js-cookie",
+  "repo": "js-cookie/js-cookie",
+  "files": {
+    "include": ["src/js.cookie.js"]
```

Comment:
> Is there any chance you could add the min file somewhere?

The minified file is available as an attachment on each github release ([example](https://github.com/js-cookie/js-cookie/releases/tag/v2.0.2)).